Libraries/LibWeb/CSS/ComputedProperties.h

```diff
@@ -283,7 +282,6 @@ class WEB_API ComputedProperties final : public JS::Cell {
     RefPtr<Gfx::Font const> m_first_available_computed_font;
 
     Optional<CSSPixels> m_line_height;
-    Optional<CSSPixels> m_font_size;
```

Comment:
We originally added this because `font_size()` was getting called all over the place in layout (due to units like `em` resolving against the font size). It seems likely that changing this to a `property()` lookup every time would cause a performance regression :)

Reply:
Ah you are right! I misremembered :)